Tools/ssl/multissltests.py

```diff
@@ -18,12 +18,15 @@
 search paths for header files and shared libraries. It's known to work on
 Linux with GCC and clang.
 
-Please keep this script compatible with Python 2.7, and 3.4 to 3.7.
+Please keep this script compatible with all currently-maintained Python
+versions.
```

Comment:
Do we care about Python 3.9 at this point? Frequently for in-tree tools we use python_for_regen as the base version, or the previous stable release (e.g. 3.12 at present).

Reply:
True, but e.g. for backports to 3.9 we would run against the version of `Tools/ssl/multissltests.py` in the 3.9 branch? We should add a comment explaining why we care about all non EOL versions, though.